ivy/functional/backends/paddle/experimental/elementwise.py

```diff
@@ -200,7 +210,22 @@ def nan_to_num(
     neginf: Optional[Union[float, int]] = None,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    if x.dtype == paddle.float32:
```

Comment:
I don't prefer the use of static limits, instead you should use `ivy.iinfo, ivy.finfo` to optain the upper and lower bound for each data type